src/app/components/SearchResults/ResultsListItem.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Link } from 'react-router';
+import PropTypes from 'prop-types';
+import ResultsRow from './ResultsRow';
+
+/**
+ * ResultsList presents search results as a "grouped" list of books
+ * with their associated editions provided by the ResultsRow component.
+ * Each result displays a title and author element linked to its companion
+ * detailed view.
+ *
+ * @returns {string|null}
+ */
+class ResultsListItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.props = props;
+
+    this.showWorkDetail = this.showWorkDetail.bind(this);
+  }
+
+  showWorkDetail(event, workId) {
+    event.preventDefault();
+
+    this.props.fetchWork(workId)
+      .then(() => {
+        this.context.router.push(`/work?workId=${workId}`);
+      });
+  }
+
+  render() {
+    const result = this.props.item;
+    return (
+      <li className="nypl-results-item">
+        <h3>
+          <Link onClick={event => this.showWorkDetail(event, result.uuid)} to={{ pathname: '/work', query: { workId: `${result.uuid}` } }}>
+            {result.title}{(result.entities && Array.isArray(result.entities)) ? ` – ${result.entities[0].name}` : ''}
```

Comment:
Use `isArray` from `underscore` or `.length` as a test instead of `Array.isArray`? Well...this is going to change based on the designs and from the UX document modules. This is the pattern used by our Design Toolkit (title - author)

Reply:
done